mono/metadata/w32file-unix.c

```diff
@@ -3103,6 +3105,8 @@ mono_w32file_set_times(gpointer handle, const FILETIME *create_time, const FILET
 					 write_time));
 }
 
+#endif
```

Comment:
I've removed the code that tries to build the IL2CPP on Mono static libraries now, so we should be able to remove this #if wrapper.